parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -930,19 +945,70 @@ public ParquetFileReader(InputFile file, ParquetReadOptions options) throws IOEx
     this(file, options, file.newStream());
   }
 
+  /**
+   * @param file    Path to a parquet file
+   * @param options {@link ParquetReadOptions}
+   * @param f       a {@link SeekableInputStream} for the parquet file
+   * @throws IOException if the file can not be opened
+   */
   public ParquetFileReader(InputFile file, ParquetReadOptions options, SeekableInputStream f) throws IOException {
     this.converter = new ParquetMetadataConverter(options);
     this.file = file;
     this.f = f;
     this.options = options;
     try {
       this.footer = readFooter(file, options, f, converter);
-    } catch (Exception e) {
+    } catch (IOException e) {
```

Comment:
why change it?

Reply:
it was not intentional. thanks for catching it.